demisto_sdk/commands/lint/lint_manager.py

```diff
@@ -152,7 +153,13 @@ def _get_packages(self, content_repo: git.Repo, input: str, git: bool, all_packs
         elif not all_packs and not git and not input:
             pkgs = [Path().cwd()]
         else:
-            pkgs = [Path(item) for item in input.split(',')]
+            pkgs = []
+            for item in input.split(','):
+                if os.path.isdir(item) and os.path.exists(os.path.join(item, PACKS_PACK_META_FILE_NAME)):
```

Comment:
I'd add a line with `is_pack=CONDITION` and then `if is_pack:` for readability, your call